ts/features/zendesk/saga/index.ts

```diff
@@ -79,3 +99,54 @@ export function* watchZendeskSupportSaga() {
     dismissSupport();
   });
 }
+/**
+ *
+ * @param getSession is the API call to get the session tokens.
+ * The goal of this saga is to take Zendesk token from the BE in order to properly report to support.
+ */
+function* getZendeskTokenSaga(
+  getSession: ReturnType<typeof BackendClient>["getSession"]
+) {
+  try {
+    // Define the fields needed for the token request, in this case, the needed field is only 'zendeskToken'
+    const fields = formatRequestedTokenString(false, ["zendeskToken"]);
+    const isFastLogin = yield* select(isFastLoginEnabledSelector);
+
+    const response = (yield* call(
+      withRefreshApiCall,
+      getSession({ fields }),
+      getZendeskToken.failure("401") // if the error is 401 the error screen is not show thanks this parameter
+    )) as unknown as SagaCallReturnType<typeof getSession>;
```

Comment:
```suggestion
    const response = (yield* call(
      withRefreshApiCall,
      getSession({ fields }),
      getZendeskToken.failure("401") // if the error is 401 the error screen is not show thanks this parameter
    )) as SagaCallReturnType<typeof getSession>;
```
The first cast to `unknown` is necessary? What about removing it and directly cast as in this suggestion? Could it work?

Reply:
Thanks for this suggestion! I've made the change🚀